src/extraction/parseHardString.ts

```diff
@@ -1,3 +1,4 @@
+import { Config } from '~/core'
```

Comment:
Test a failing with an error:
```
Error: Cannot find module '~/core'
Require stack:
- /Users/terales/Documents/GitHub/i18n-ally-lokalise/src/extraction/parseHardString.ts
```

It looks like you can't import it in this way and I'm not sure not how exactly it should be done

Reply:
![image](https://github.com/lokalise/i18n-ally/assets/46187796/4fbee9b2-f2c7-4ca8-b9f0-793245bf7380)
how about fix in this way, add a ` getString` param to the `parseHardString`, then find all the usage, and set the ` getString` param to 
```
(args) => `${Config.argsPrefix}${args.length - 1}${Config.argsSuffix}`
```

Reply:
Lets try and see how it look,
just call it an `argsToStringCallback` please

Reply:
all unit test  passed, I'll push this commit